qemu/tests/boot_with_machine_types.py

```diff
@@ -31,14 +30,14 @@ def run(test, params, env):
     if not machine_types:
         test.fail("Failed to get machine types")
     else:
-        logging.info("Actual supported machine types are: %s", ', '.join(map(str, machine_types)))
+        test.log.info("Actual supported machine types are: %s", ', '.join(map(str, machine_types)))
```

Comment:
more than 79 chars

Reply:
Per our discussion, style issues will not be fixed this time, thanks!